FlowCrypt/src/main/java/com/flowcrypt/email/security/pgp/PgpMsg.kt

```diff
@@ -1008,8 +1008,15 @@ object PgpMsg {
     return when (msgBlock) {
       is GenericMsgBlock -> {
         when (msgBlock.type) {
-          MsgBlock.Type.PLAIN_HTML -> msgBlock.copy(type = MsgBlock.Type.DECRYPTED_HTML)
-          MsgBlock.Type.PLAIN_TEXT -> msgBlock.copy(type = MsgBlock.Type.DECRYPTED_TEXT)
+          MsgBlock.Type.PLAIN_HTML -> msgBlock.copy(
+            type = MsgBlock.Type.DECRYPTED_HTML,
+            content = msgBlock.content?.let { stripFcReplyToken(it) }
+          )
+
+          MsgBlock.Type.PLAIN_TEXT -> msgBlock.copy(
+            type = MsgBlock.Type.DECRYPTED_TEXT,
+            content = msgBlock.content?.let { stripFcReplyToken(it) })
+
```

Comment:
It's strange, but my build issue was fixed by moving closing bracket to the separate line, so
```kt
content = msgBlock.content?.let { stripFcReplyToken(it) })
```
became
```kt
  content = msgBlock.content?.let { stripFcReplyToken(it) }
)
```

Can you please update it here too

Reply:
Sure. Updated!